src/application.js

```diff
@@ -422,6 +422,14 @@ class XtkSchemaNode {
          */
         this.nodePath = this._getNodePath(true)._path;
 
+        if (this.isRoot != undefined) {
+          this._buildLocalizationIds();
+        } else {
```

Comment:
It seems that _buildLocalizationIds would work for both cases where this.isRoot is defined or not. It's not clear why isRoot would be undefined rather that true or false.

Reply:
it seems isRoot is undefined when it is the label of the schema, it is the case of XtkSchema